docs/docs/sourcing-from-gentics-mesh.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Sourcing from Gentics Mesh
+---
+
+## Using Gentics Mesh
+
+[Gentics Mesh](https://getmesh.io) is an open-source self-hosted headless CMS that supports GraphQL. Contents like posts, products, images and more are represented by elements called "nodes". These nodes can then be queried with the familiar GraphQL syntax.
+
+The model of nodes can be defined within "schemas". In our demo we have schemas for vehicles, images and categories.
```

Comment:
Please try not to use "we" (https://www.gatsbyjs.org/contributing/gatsby-style-guide/#use-you-as-the-pronoun).

You could rephrase the sentence to make it more clear that with "we" Gentics Mesh is meant and that Gentics Mesh provides this demo etc.

Reply:
Good point. I have rephrased the sentence.